tests/Support/TestHelperTrait.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Tests\Support;
+
+use App\Record;
+use App\RecordIssuer;
+use App\RecordIssuerType;
+use App\User;
+use Illuminate\Contracts\Console\Kernel;
+use Mockery\CountValidator\Exception;
+
+trait TestHelperTrait
+{
+    // return s array of users
+    private function generateUsersInDatabase($count = 1) {
+        return factory(User::class, $count)->create();
+    }
+
+    private function getBillOrgTypeId() {
+        return RecordIssuerType::type(RecordIssuerType::BILLORG_TYPE_NAME)->first()->id;
+    }
+
+    private function createRecordIssuerModel($organizationName) {
```

Comment:
This is just the same as `factory(RecordIssuer::class)->make([override attributes...])`